### DISCUSSION TITLE
Output Delay Setting in MPD

### DISCUSSION THREAD

**Discussion by tortuga-Bob** [MusicPlayerDaemon/MPD#2004] [General]
Is there a configuration setting in MPD to allow an output delay when changing sample rates? This is also referred to as resync delay in Roon-speak. This gives the DAC extra time to lock to the new sample rate stream and prevents cutting off the beginning of a track that has no lead-in time.  My searches have not revealed such a config setting, is this correct? Is this something that can be added? Thanks!

**Comment by kallekutta** [comment]
https://github.com/MusicPlayerDaemon/MPD/issues/1312
